hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/utils/MetadataConversionUtils.java

```diff
@@ -137,6 +141,41 @@ public static HoodieArchivedMetaEntry createMetaWrapper(HoodieInstant hoodieInst
     return archivedMetaWrapper;
   }
 
+  public static HoodieArchivedInstant createArchivedInstant(ActiveInstant activeInstant, HoodieTableMetaClient metaClient) {
```

Comment:
add UT

Reply:
Already added in `TestMetadataConversionUtils`.